### PR TITLE
New home design

### LINKED ISSUE #4669: Graphic updates for the Home page
We wanted to add some diagrams and icons to the home page.

Here is the guidance for the design provided by the contractor Lionsway we worked with.
[Home Page Design Updates Jan-2022.pdf](https://github.com/knative/docs/files/7936851/Home.Page.Design.Updates.Jan-2022.pdf)

And they also provided the graphics assets
[Knative Web Assets.zip](https://github.com/knative/docs/files/7936849/Knative.Web.Assets.zip)

### PR DESCRIPTION
**Pull request by gabo1208**
Fixes #4669
Implement home page design with new graphics provided

### CODE REVIEW

**Comment by snneji** [comment]
Looking good so far - thanks @gabo1208!

**Comment by csantanapr** [comment]
@gabo1208 
Can you make the whole square of use case study clickable?
image

**Comment by csantanapr** [comment]
Can you make the button "Deploy Knative Today" go to Installing
https://deploy-preview-4715--knative.netlify.app/development/install/
image

**Comment by csantanapr** [comment]
Add the word "Search" to the search box like the current one
image

image

File: overrides/home.html [hunk 1]
```diff
@@ -1,157 +1,289 @@
...
-    </div>
     <div class="md-grid md-typeset">
       <div class="mdx-hero">
-        <div>
-          <h1>Enterprise-grade Serverless on your own terms.</h1>
-          <h2>Kubernetes-based platform to deploy and manage modern serverless workloads.</h2>
+        <div class="headline-section">
+          <h1 class="main-headline">
+            Knative is an open-source enterprise-level<br>
+            solution for leveraging Kubernetes in<br>
```
Comment (csantanapr) [inline overrides/home.html:11]:
```suggestion
            solution for leveraging Kubernetes<br>
```

File: overrides/home.html [hunk 2]
```diff
@@ -1,157 +1,289 @@
...
-
-        <a href="{{ 'about/case-studies/puppet/' | url }}">
-        <div class="logo">
-          <img src="images/case-studies/puppet.png" alt="" draggable="false" />
+  <section class="knative-components-container">
+    <div class="md-grid md-typeset">
+      <h1>Knative Components</h1>
+      <p class="normal-text">
+        Knative has two main components that empower teams working with Kubernetes. Serving and<br>
+        Eventing work together to automate and manage tasks and applications.
```
Comment (csantanapr) [inline overrides/home.html:159]:
```suggestion
        Eventing works together to automate and manage tasks and applications.
```

File: overrides/home.html [hunk 3]
```diff
@@ -1,157 +1,289 @@
...
+            </div>
+          </div>
+          <div class="img-col">
+            <div class="reason-img">
+              <img src="images/home-images/blue_plugable_icon.png"
+                onmouseover="this.src='images/home-images/dkblue_plugable_icon.png'"
+                onmouseout="this.src='images/home-images/blue_plugable_icon.png'" alt="" draggable="false">
+            </div>
+            <div class="reason-text">
+              <h2>Plugable</h2>
```
Comment (csantanapr) [inline overrides/home.html:143]:
```suggestion
              <h2>Pluggable</h2>
```

**Review by csantanapr — COMMENTED** [review]
Great work, just some minor things

**Comment by aijamalnk** [comment]
This is really awesome @gabo1208 . Thank you very much for contributing it! LGTM from my side.

**Comment by gabo1208** [comment]
Thanks all :)!! And thanks for the reviews <3

**Comment by csantanapr** [comment]
FYI:
holding on an icon to be provided next Monday
/hold

**Comment by csantanapr** [comment]
@gabo1208 here is the missing icon
dkblue_autoscaling_icon

**Comment by gabo1208** [comment]
Ready to merge!

**Comment by snneji** [comment]
@n3wscott, I don't think so. I believe this PR was just to redesign the layout of the home page. The logo looks the same to me on the website. Where can you see a change?

**Comment by gabo1208** [comment]
Fixed in #4800

**Pull request merged**